src/layouts/index.js

```diff
@@ -22,31 +25,111 @@ injectGlobal`
   }
 `;
 
+class AnimatedScreen extends React.Component {
+  wrapper = React.createRef();
+
+  animate = () => {
+    const wrapper = this.wrapper.current;
+
+    this.animation = new TimelineLite();
+    this.animation
+      .set(wrapper, { x: '-100%' })
+      .to(wrapper, 0.4, { x: '0%' })
+      .to(wrapper, 0.4, { x: '-100%' }, '+=0.4')
+      .set(wrapper, { x: '-100%' });
+    this.animation.play();
+  };
+
+  render() {
+    return (
+      <Box
+        innerRef={this.wrapper}
+        bg={'sea'}
+        style={{
```

Comment:
Tu peux par exemple étendre `Box` pour faire ça en styled-component : 
```
const AniamtedScreenWrapper = Box.extend`
  position : absolute;
  etc
  `;
```
et même faire un nouveau composant 'wrapper' parce qu'en fait on ne se sert pas des props de `Box` !!

Ou encore rajouter les props de positionnement au componsant `Box` 
https://github.com/jxnblk/styled-system/blob/master/docs/api.md#position

```
export const Box = system(
  { blacklist: ['overflowHidden'] },
  'space',
  'width',
  'color',
  'textAlign',
  'position',
  'top',
  'etc
);
```

Reply:
Tu préfères quoi ?

Reply:
ouais le plus approprié c'est peut-être : 
```
const AnimatedScreenWrapper = Box.extend`
  ${absolute({top:0, bottom:0, left: 0, right:0});
  z-index: 999;
  transform: 'translateX(-100%)'
 `;
```
puis `<AnimatedScreenWrapper  innerRef={this.wrapper} bg='sea'>`

Reply:
ça coûte quoi de rajouter les props à Box ?  
ça me va d'extend Box pour ce cas mais pour le ViewWrapper qui fait juste position absolute et width 100%, ça devient lourd de déclarer un nouveau composant je trouve.